rpcadaptors.go

```diff
@@ -464,3 +465,39 @@ var _ rpcserver.FeeEstimator = (*rpcFeeEstimator)(nil)
 type rpcFeeEstimator struct {
 	*fees.Estimator
 }
+
+// Ensure rpcBlockTemplater implements the rpcserver.Templater interface.
+var _ rpcserver.BlockTemplater = (*rpcBlockTemplater)(nil)
+
+// rpcBlockTemplater provides a block template generator for use with the
+// RPC server and implements the rpcserver.BlockTemplater interface.
+type rpcBlockTemplater struct {
+	g *mining.BgBlkTmplGenerator
```

Comment:
Given the interface signatures match, this should be embedded so that it just automatically implements the interface without having to specify all the funcs again.